test/e2e/tests/account/forgot-password.spec.ts

```diff
@@ -30,6 +30,8 @@ describe('Forgot password', function () {
         // Lock Wallet
         const homePage = new HomePage(driver);
         await homePage.headerNavbar.check_pageIsLoaded();
+        // We need a bigger delay before locking the wallet to avoid the error '#snapGetPublicKey - unable to proceed, wallet is locked'
+        await driver.delay(5000);
```

Comment:
this spec also has the same problem, so fixing it in the same branch, to make it easier to merge and not being kicked out from the queue